### DISCUSSION TITLE
Configurar hora (zona horaria) en Mirth Connect

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5402] [Q&A] [answered]
Buen dia, podrian orientarme en como configurar la zona hoaria o cambiar la hora.
No me refiero a la hora que se asigna la ejecucion de los canales, sino la hora de forma general.
Aca se cambio la hora en Chile, por ende se desconfiguro todo, y los canales se ejecutan una hora antes de lo programado.

Quedo atenta, saludos.

**Comment by pacmano1** [comment, marked as answer]
See https://forums.mirthproject.io/forum/mirth-connect/general-discussion/181983-configurar-zona-horaria-en-mc.  

If you cross post here and the forums, please reference that fact 

Answer at forum link above.
